### PR TITLE
Clear all feature

### PR DESCRIPTION
**Pull request by giriputraadhittana01**
clear all trash

### CODE REVIEW

**Comment by giriadhittana01** [comment]
> clear all trash

Hello

File: main.go [hunk 1]
```diff
@@ -9,11 +9,4 @@ func main() {
 	fmt.Println("Hello World")
 	fmt.Println("New Feature")
 	greeting("Giri Putra Adhittana")
-	fmt.Println("New Feature B")
-	fmt.Println("New Feature C")
-	fmt.Println("New Feature C - hello")
-	fmt.Println("New Feature D")
```
Comment (giriadhittana01) [inline main.go:15]: This line not supposed to remove

Comment (giriadhittana01) [reply main.go:15]: only that

Comment (giriputraadhittana01) [reply main.go:15]: okay

Comment (giriputraadhittana01) [reply main.go:15]: so ?

Comment (giriputraadhittana01) [reply main.go:15]: what?

**Pull request merged**